README.md

```diff
@@ -253,13 +253,13 @@ Please ensure you have the required [CUDA dependency](#cuda-dependency).
 
 The Common Voice corpus consists of voice samples that were donated through Mozilla's [Common Voice](https://voice.mozilla.org/) Initiative.
 
-We provide an importer (`bin/import_cv.py`) which automates downloading and preparing the Common Voice corpus as such:
+We provide an importer (`bin/import_cv2.py`) which automates downloading and preparing the Common Voice (v2.0) corpus as such:
```

Comment:
This is not true.

It does not automate downloading. `import_cv.py` automated downloading `import_cv2.py` does not due to the GDPR.

Reply:
12d31c11bde9b214b703e0cd44cfb9e3ce0b14a8